crates/bevy_render/src/camera/projection.rs

```diff
@@ -549,7 +553,7 @@ impl OrthographicProjection {
         OrthographicProjection {
             scale: 1.0,
             near: 0.0,
-            far: 1000.0,
+            far: Some(Self::DEFAULT_FAR_PLANE),
```

Comment:
I feel like this should default to None since bevy currently doesn't do far plane culling.

Reply:
This is the orthographic case, where bevy does currently do far culling. See https://github.com/bevyengine/bevy/issues/16746 (Additional information)

Reply:
Does `None` even make sense for ortho? It doesn't have the ability to use infinite z, because depth is linear.